docs/geozarr-minispec.md

```diff
@@ -226,9 +226,16 @@ In this case `lat` is both a coordinate variable and a data variable.
 
 ## Multiscale Dataset 
 
+This implementation supports **two multiscales metadata conventions**:
+
+1. **[Zarr Multiscales Convention](https://github.com/zarr-conventions/multiscales)**: An experimental convention for describing multi-resolution data with simple scale and translation metadata
+2. **GeoZarr 0.4 TileMatrixSet Specification**: The experimental GeoZarr specification using OGC TileMatrixSet definitions for geospatial data
+
+Both conventions can be generated simultaneously, providing flexibility for different use cases and ensuring compatibility with various tools and workflows.
```

Comment:
Does this mean, in the same zarr you can have the two types of multiscale?

Could we have a simple  zarr.json example of this scenario?